include/seqan/arg_parse/arg_parse_type_support.h

```diff
@@ -71,6 +93,23 @@ inline TTarget _cast(TString const s)
 }
 
 // ----------------------------------------------------------------------------
+// Function _castBool()
+// ----------------------------------------------------------------------------
+
+template <typename TString>
+inline bool _castBool(bool & dst, TString const s)
+{
+    // since the validValue check already verifies that the value if one of
```

Comment:
> One would have to add a special case only for boolean values which I think is not consistent with the general use of the argument parser.

Well at the other place it also says in the code:
```c+++
// we only check valid values for files and string arguments, but not for prefix arguments
```
So it's the perfect place to introduce a transform for bools and changing the comment or not? :laughing: